blocks/single-column-layout-block/layouts/single-column-regular/index.story.jsx

```diff
@@ -23,22 +23,50 @@ const Navigation = () => <div style={styles}>Navigation</div>;
 const Component = ({ children }) => <div style={styles}>{children}</div>;
 const Footer = () => <div style={styles}>Footer</div>;
 
-export const layoutWithOneChild = () => (
+export const oneChildInBody = () => (
```

Comment:
Looking at this in Storybook it's showing the Footer above the body which is in correct based on what we want
<img width="319" alt="Google Chrome-2022-06-09-14-26 03" src="https://user-images.githubusercontent.com/868127/172858146-90fb6239-8222-44c4-b787-f258909c37ab.png">